exercises/pig-latin/description.md

```diff
@@ -5,12 +5,10 @@ confusing. It obeys a few simple rules (below), but when it's spoken
 quickly it's really difficult for non-children (and non-native speakers)
 to understand.
 
-* **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to
-  the end of the word. Treat "xr" and "yt" at the beginning of a word as vowels ("xray" -> "xrayay").
-* **Rule 2**: If a word begins with a consonant sound, move it to the
-  end of the word, and then add an "ay" sound to the end of the word.
-* **Rule 3**: If a word starts with the following consonant clusters "ch", "qu", "th", "thr", "sch" and any consonant followed by "qu" at the beginning of the word, move it to the end of the word, and then add an "ay" sound to the end of the word (e.g. "chair" -> "airchay", "square" -> "aresquay", "thread" -> "eadthray").
-* **Rule 4**: If "y" comes as the beginning of the word then it must be treated as a consonant, and if it comes after a consonant cluster or is the second letter in a two letter word it must be treated as a vowel.
+- **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to the end of the word. Treat "xr" and "yt" at the beginning of a word as vowels (e.g. "xray" -> "xrayay", "yttria" -> "yttriaay").
+- **Rule 2**: If a word begins with a consonant sound (can be made up of multiple consonants a.k.a consonant cluster), move it to the end of the word.
+- **Rule 3**: If a word starts any consonant followed by "qu" at the beginning of the word, move it to the end of the word, and then add an "ay" sound to the end of the word (e.g. "square" -> "aresquay").
+- **Rule 4**: If a word contains a "y" after a consonant cluster or as the second letter in a two letter word treat it as a vowel by leaving in it place.
 
 There are a few more rules for edge cases, and there are regional
 variants too.
```

Comment:
I am not convinced of the relevancy of this statement.  Perhaps if it was also stated that this exercise is focused on the 4 rules described above it would increase the relevance of the statement....  I'm interested to know what other people think.

Reply:
True, as the 4 rules pretty much explain how to make the tests pass.

Reply:
> True, as the 4 rules pretty much explain how to make the tests pass.

Or better stated, how Pig Latin works.

Reply:
Yes!

Reply:
I kind of like having true statements included in a problem README, even if strictly speaking they're not relevant to solving the exercise. Additional context can mean the difference between a tedious exercise and an interesting problem.

Many people grew up speaking pig latin as children, and they may notice variations between what they remember and what is tested. For example, in the pig latin of my own childhood, "skylight" would be rendered "aylightskay" or "i-lightskay", which isn't covered by the four rules as listed here. Leaving in the note about edge case and regionalisms costs nothing and is validating.